.gitignore

```diff
@@ -1,3 +1,4 @@
 # Ignore Mac DS_Store files
 .DS_Store
 **/.DS_Store
+.idea
```

Comment:
This is not needed

Reply:
It is needed to ignore IntelliJ idea project files.
If you could add this It would make my live easier.

Reply:
I don't have a problem adding it to the code. But it's not related to this PR. 

Maybe we could create a new PR and also ignore files for VS Code

Reply:
I removed it